src/System.Private.CoreLib/src/System/Diagnostics/Eventing/DotNETRuntimeEventSource.cs

```diff
@@ -6,19 +6,19 @@ namespace System.Diagnostics.Tracing
 {
 #if FEATURE_PERFTRACING
     /// <summary>
-    /// RuntimeEventSource is an EventSource that represents the ETW/EventPipe events emitted by the native runtime.
-    /// Most of RuntimeEventSource is auto-generated by scripts/genRuntimeEventSources.py based on the contents of the Microsoft-Windows-DotNETRuntime provider.
+    /// NativeRuntimeEventSource is an EventSource that represents the ETW/EventPipe events emitted by the native runtime.
+    /// Most of NativeRuntimeEventSource is auto-generated by scripts/genRuntimeEventSources.py based on the contents of the Microsoft-Windows-DotNETRuntime provider.
```

Comment:
Could you please rename the file to `NativeRuntimeEventSource.cs`. Our convention is that the type name should match the file name.